src/client/pages/AssessmentSection/Descriptions/CommentableDescription/Description/DataSources/DataSourceColumn/ColumnComments.tsx

```diff
@@ -6,24 +6,25 @@ import { DataSource } from '@meta/assessment'
 
 import DataColumn from '@client/components/DataGrid/DataColumn'
 import VerticallyGrowingTextField from '@client/components/VerticallyGrowingTextField'
-import { datasourceValidators } from '@client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSourceColumn/DataSourceColumn'
+
+import { datasourceValidators } from './datasourceValidators'
 
 type Props = {
   disabled: boolean
-  dataSource: DataSource
+  dataSourceValue: DataSource
```

Comment:
Note: Added more descriptive names for dataSourceValue (= eg. assessment_fra_2025.descriptions entity) and dataSourceMetadata (= eg. assessment_fra.section.props.descriptions... entity)